addons/sys_intercom/XEH_postInit.sqf

```diff
@@ -61,19 +61,16 @@ if (!hasInterface) exitWith {};
     [FUNC(enterVehicle), [_newVehicle, _player]] call CBA_fnc_execNextFrame; // Make sure vehicle info UI is created
 }, true] call CBA_fnc_addPlayerEventHandler;
 
+// Hide display when entering a feature camera
 ["featureCamera", {
     params ["_player", "_featureCamera"];
     if (_featureCamera isEqualTo "") then {
-        [FUNC(enterVehicle), [vehicle _player, _player]] call CBA_fnc_execNextFrame; // Make sure vehicle info UI is created
+        [vehicle _player, _player] call FUNC(updateVehicleInfoText); // Update & Show
+    } else {
+        [false] call EFUNC(sys_gui,showVehicleInfo); // Hide
     };
 }, true] call CBA_fnc_addPlayerEventHandler;
 
-player addEventHandler ["seatSwitchedMan", {
```

Comment:
What happens then when switching seats? How is it captured in cargo seats?

Reply:
Intercom PFH is the source of all information. It has to run anyways, so I capture seat changes there by already existing functions you wrote.

This method was poor, it only works on `player` (no remote control) and only  for vanilla switching (no ACE "Change seat" or scripted). New method captures all of it, including switching between specific cargo seats, to comply with the configuration ability which can do the same.

Reference last commit for that change.

Reply:
is the frequency of the PFH enough or should it run faster?